conda-forge.yml

```diff
@@ -1,5 +1,10 @@
-channel_priority: strict
-azure:
-  settings_win:
-    pool:
-      vmImage: windows-2022
+build_platform:
+  linux_aarch64: linux_64
+  linux_ppc64le: linux_64
+  osx_arm64: osx_64
+conda_build:
+  error_overlinking: true
+conda_forge_output_validation: true
+github:
+  branch_name: main
+  tooling_branch_name: main
```

Comment:
you should not edit this feedstock's file.

rather you need to create this file within the directory

`recipes/xsel`

Reply:
Oh. OK.

Reply:
That was what @xhochy said to me earlier.

Reply:
Fixed.

Reply:
lets see, its kinda a gamble if it works, but you are trying to get quite a few feedstocks in, so it might be helpful